API.md

```diff
@@ -0,0 +1,552 @@
+# API
+
+* [Static Functions](#static-functions)
+  * [`create`](#create)
+* [Instance Methods](#instance-methods)
+  * [`start`](#start)
+  * [`stop`](#stop)
+  * [`dial`](#dial)
+  * [`dialProtocol`](#dialProtocol)
+  * [`hangUp`](#hangUp)
+  * [`handle`](#handle)
+  * [`unhandle`](#unhandle)
+  * [`peerRouting.findPeer`](#peerRouting.findPeer)
+  * [`contentRouting.findProviders`](#contentRouting.findProviders)
+  * [`contentRouting.provide`](#contentRouting.provide)
+  * [`_dht.put`](#_dht.put)
+  * [`_dht.get`](#_dht.get)
+  * [`_dht.getMany`](#_dht.getMany)
```

Comment:
If these are not for public consumption they should not be included in the docs.

Reply:
I understand that! 

However, taking into account that project such as `js-ipfs` use these API methods, I also don't like that we do not document something that will probably be widely used (also discussed previously on https://github.com/libp2p/js-libp2p/pull/472#discussion_r345011828)

Reply:
I am updating to `contentRouting`